packages/yourloops/components/dashboard-widgets/patient-dashboard.tsx

```diff
@@ -147,13 +158,30 @@ const PatientDashboard: FunctionComponent<PatientDashboardProps> = (props) => {
           onSwitchToDaily={onSwitchToDaily}
         />
       </Grid>
-      {showRemoteMonitoringWidget &&
-        <Grid item xs={gridWidgetSize}>
+
+      {patientIsMonitored &&
+        <React.Fragment>
+          <Grid item xs={gridWidgetSize} className={classes.gridItemContainer}>
+            <AlarmCard patient={patient} />
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏All these shenanigans make me think that the patient hook could have a state `selectedPatient`, that way we would not have to pass the patient down like that :slightly_smiling_face: 
This would be a nice future improvment!